deploy/kube-bench/templates/daemonset.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-bench-ds
+spec:
+  selector:
+    matchLabels:
+      app: kube-bench
+  template:
+    metadata:
+      labels:
+        app: kube-bench
+      name: kube-bench
+    spec:
+      hostPID: true
+      containers:
+        - name: kube-bench
+          image: {{ .Values.image.repository }}/{{ .Values.image.name }}:{{ .Values.image.tag }}
+          imagePullPolicy: IfNotPresent
```

Comment:
isn't it better to remove this and let kubernetes put it's default (always for latest and ifnotpresent else)